frontend/src/metabase/notifications/pulse/components/ChannelCard.tsx

```diff
@@ -0,0 +1,34 @@
+import cx from "classnames";
+
+import CS from "metabase/css/core/index.css";
+import { Flex, Icon, Paper, Text } from "metabase/ui";
+
+export const ChannelCard = ({
+  onClick,
+  title,
+  channel,
+}: {
+  onClick?: () => void;
+  title: string;
+  channel: "email" | "slack";
+}) => {
+  const iconName = channel === "email" ? "mail" : "slack_colorized";
+
+  return (
+    <Paper
+      withBorder
+      radius="md"
+      p="md"
+      shadow="none"
+      onClick={onClick}
+      className={cx(CS.cursorPointer, CS.bgLightHover, CS.textBrandHover)}
```

Comment:
I really appreciate that we have these classes. They were a perfect match for this particular use case.